Source/Core/Common/bbaserver.h

```diff
@@ -0,0 +1,64 @@
+// Copyright 2019 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <QObject>
+#include <QSet>
+
+class QHostAddress;
+class QNetworkProxy;
+class QTcpServer;
+
+class BbaClient;
+class BbaDebug;
+
+class BbaServer : public QObject
+{
+  Q_OBJECT
+
+public:
+  explicit BbaServer(QObject *parent = Q_NULLPTR);
```

Comment:
```suggestion
  explicit BbaServer(QObject *parent = nullptr);
```

There's no need to use Q_NULLPTR given we require a C++14-capable compiler.